Tests/MetricsTests/TestMetrics.swift

```diff
@@ -36,7 +36,10 @@ internal final class TestMetrics: MetricsFactory {
     }
 
     public func makeTimer(label: String, dimensions: [(String, String)]) -> TimerHandler {
-        return self.make(label: label, dimensions: dimensions, registry: &self.timers, maker: TestTimer.init)
+        let maker = { (label: String, dimensions: [(String, String)]) -> TimerHandler in
+            TestTimer(label: label, dimensions: dimensions)
+        }
+        return self.make(label: label, dimensions: dimensions, registry: &self.timers, maker: maker)
```

Comment:
i think this is not required since ctor signature did not change?

Reply:
Ah, yes. I'll revert it to what it was before!